search-commons/src/main/java/no/unit/nva/utils/IndexDocumentWrapperLinkedData.java

```diff
@@ -0,0 +1,56 @@
+package no.unit.nva.utils;
+
+import no.unit.nva.search.IndexDocument;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.ioutils.IoUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static nva.commons.apigateway.MediaTypes.APPLICATION_JSON_LD;
+
+@JacocoGenerated
+public class IndexDocumentWrapperLinkedData {
+
+    private static final String EMPTY_STRING = "";
+
+    private final UriRetriever uriRetriever;
+
+    public IndexDocumentWrapperLinkedData(UriRetriever uriRetriever) {
+        this.uriRetriever = uriRetriever;
+    }
+
+    public String toFramedJsonLd(IndexDocument indexDocument) {
+        try (InputStream frame = new SearchIndexFrame().asInputStream()) {
+            return new FramedJsonGenerator(getInputStreams(indexDocument), frame).getFramedJson();
+        } catch (InterruptedException | IOException e) {
+            e.printStackTrace();
```

Comment:
We should not use printStackTrace. We can use the logger and we can also use ExceptionUtils from nva-commons.core to printit in one line. 
The question is since each document is indexed in a separate process what is the purpose of marking an unsuccessful indexing action as successful by catching the exception? Why can't we just throw it?